packages/core/src/scuttle.js

```diff
@@ -22,6 +22,14 @@ const {
   defineProperty,
 } = Object
 
+const avoidForLavaMoatCompatibility = (function(cg, g) {
```

Comment:
Don't like the name, disagree with what it says. 🫣

I'd call it `preserveLanguageIntrinsics` or something like that

Reply:
fixed